### PR TITLE
Add search box where the PDF and "Get the Code" link was

### LINKED ISSUE #2: Find search
Can we add a "find" search request to query the database for both facility and code names?

### PR DESCRIPTION
**Pull request by radford-for-smpte**
Closes #2 

Note: I'm not sure sorting by column headings is even necessary given this search box.

### CODE REVIEW

File: src/main/templates/facilities.hbs [hunk 1]
```diff
@@ -31,10 +25,8 @@
     <tbody>
       {{#data}}
       <tr>
-        <td>
-          <pre>{{code}}</pre>
```
Comment (radford-for-smpte) [inline src/main/templates/facilities.hbs:35]: I found that the pre tags caused alignment problems in the table. Additionally, I don't think they're necessary as handlebars handles HTML quoting.

File: src/main/templates/facilities.hbs [hunk 2]
```diff
@@ -7,19 +7,13 @@
...
 
 <body>
   <nav class="navbar navbar-expand-lg navbar-light bg-light justify-content-between d-print-block">
     <a class="navbar-brand" href="#">ISDCF Facilities List</a>
-    <ul class="navbar-nav d-print-none">
-      <li class="nav-item">
-        <a class="nav-link" href="https://github.com/isdcf/registries">Get the code</a>
-      </li>
-      <li class="nav-item">
-        <a class="nav-link" href="{{pdf_path}}">Download as PDF</a>
```
Comment (palemieux) [inline src/main/templates/facilities.hbs:20]: Need PDF back per #4

Comment (radford-for-smpte) [reply src/main/templates/facilities.hbs:20]: OK pdf is back.

File: src/main/templates/facilities.hbs [hunk 3]
```diff
@@ -31,10 +29,8 @@
     <tbody>
       {{#data}}
       <tr>
-        <td>
-          <pre>{{code}}</pre>
```
Comment (palemieux) [inline src/main/templates/facilities.hbs:35]: `<pre>` was intended to generate Courier text, implying that the string is the exact value to use. What about `<code>{{code}}</code>` instead?

File: src/main/templates/facilities.hbs [hunk 4]
```diff
@@ -7,21 +7,19 @@
...
+  <script type="application/javascript" src="facilities.js" defer></script>
+  <style> table td:first-child { width: 9px }</style>
 </head>
 
 <body>
   <nav class="navbar navbar-expand-lg navbar-light bg-light justify-content-between d-print-block">
     <a class="navbar-brand" href="#">ISDCF Facilities List</a>
-    <ul class="navbar-nav d-print-none">
-      <li class="nav-item">
-        <a class="nav-link" href="https://github.com/isdcf/registries">Get the code</a>
```
Comment (palemieux) [inline src/main/templates/facilities.hbs:17]: Where would the link to the github repo be -- until we have a proper service setup? #5 suggests opening a new window.

**Pull request merged**